todo-app/src/components/AddingContainer.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react"
+import { inject, observer } from "mobx-react"
+
+
+import FaPlus from "react-icons/lib/fa/plus"
+
+@inject("todosStore")
+@observer    
+export default class AddingContainer extends Component {
+   
+    handleSubmit = (e) => {
+        e.preventDefault()
+        this.props.todosStore.createTodo();
+    }
+
+    render() {
+        const { createTaskForm, editingDescription, editingDeadline } = this.props.todosStore
+        const { description } = createTaskForm 
+
+        const descriptionText = <input placeholder="enter a description" onChange={e => editingDescription(e)} />;
```

Comment:
If you are going to send **e** only as a parameter, you can do it like this `nChange={editingDescription}`